diff --git a/dpdk/config/common_linuxapp b/dpdk/config/common_linuxapp
index 0e260c2..dffba90 100644
--- a/dpdk/config/common_linuxapp
+++ b/dpdk/config/common_linuxapp
@@ -137,7 +137,7 @@ CONFIG_RTE_LIBRTE_KVARGS=y
 # Compile generic ethernet library
 #
 CONFIG_RTE_LIBRTE_ETHER=y
-CONFIG_RTE_LIBRTE_ETHDEV_DEBUG=n
+CONFIG_RTE_LIBRTE_ETHDEV_DEBUG=y
 CONFIG_RTE_MAX_ETHPORTS=64
 CONFIG_RTE_MAX_QUEUES_PER_PORT=256
 CONFIG_RTE_LIBRTE_IEEE1588=n

